kolibri/core/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import unicode_literals
+
+from django.apps import AppConfig
+
+
+class KolibriCoreConfig(AppConfig):
```

Comment:
Yeah, I think that's why other Django PRAGMA modifications I have seen have used a middleware - but I don't know of a downside to making a connection persistent.